views/default/admin/performance/generic.php

```diff
@@ -26,21 +26,23 @@
 };
 
 // apache version
-if (apache_get_version() !== false) {
-	$icon =  $icon_warning;
-	$title = elgg_echo('admin:performance:apache:mod_cache');
-	$value = elgg_echo('status:unavailable');
-	$subtext = '';
+// Check if the Server is Apache before calling apache_get_version else it may error out in server like Nginx
+if (strpos($_SERVER['SERVER_SOFTWARE'],'Apache') !== false) {	
```

Comment:
why not check on the existence of the function? Your way is (currently) case sensitive, and maybe some other webserver says it's Apache when it isn't (can happen with opensource software)

Reply:
Yeah good suggestion. I will update the code and see Elgg guidelines. That was quick dirty fix to make it working on my personal site

Reply:
> That was quick dirty fix to make it working on my personal site

understand completely ;) now try to do it the 'right' way ;)